setup.py

```diff
@@ -57,6 +59,7 @@ def build_extension(self, ext):
             build_args += ["--", "/m"]
         else:
             cmake_args += ["-DCMAKE_BUILD_TYPE=" + cfg]
+            build_args += ["--", "-j4"]
```

Comment:
Why 4?

Reply:
Just a good parallelism level since there's no reliable way to retrieve number of real cpus